cli/src/funTest/kotlin/ExamplesFunTest.kt

```diff
@@ -50,6 +50,7 @@ import org.ossreviewtoolkit.model.PackageCuration
 import org.ossreviewtoolkit.model.Severity
 import org.ossreviewtoolkit.model.config.CopyrightGarbage
 import org.ossreviewtoolkit.model.config.NotifierConfiguration
+import org.ossreviewtoolkit.model.config.PluginConfiguration
```

Comment:
This commit should refer to / auto-close #7833?

Reply:
Did this for the 2nd commit.